### DISCUSSION TITLE
2.0.0-alpha.1

### DISCUSSION THREAD

**Discussion by ruibaby** [halo-dev/halo#2500] [General]
## ⚠️ 注意

此版本为 Halo 2.0 的首个 Alpha 测试版本，**无法从 1.5 直接升级到当前版本，也不建议在生产环境使用**。当前版本仅用于测试，我们欢迎大家在测试过程中反馈任何问题。另外，在 Halo 2.0 正式发布的时候，我们会提供完整的升级指南。

## 主要特性

- 支持多用户与权限。
- 支持插件机制，将在后续的迭代过程中逐步完善文档。
- 提供 Extension API，以便插件更方便地持久化数据。
- 重新设计的主题机制，提供更加灵活地设置表单配置。
- 重新设计的附件系统，引入分组和存储策略的概念，可通过插件支持多种云存储的策略。
- 重新设计的评论系统，支持登录访问。
- 重新设计的后台管理系统。
- 默认提供富文本编辑器。
- 支持 PostgreSQL 数据库。

## 重要变更

- 无法兼容 1.x 版本的数据库，所以无法平滑升级。但在正式发布前会提供升级方案。
- 默认的模板引擎改为 Thymeleaf，所以无法兼容 1.x 的主题。Halo 官方维护的主题将逐步移植到 2.0，后续我们将提供主题开发文档以协助社区维护的主题完成移植。
- 不再内置友情链接/图库/日志功能，后续会以插件的形式提供。
- 不再内置站点地图/订阅链接的功能，后续会以插件的形式提供。
- 不再内置备份的功能，后续会以插件的形式提供。
- 不再内置云存储的功能，后续会提供常用云存储的插件。
- 不再默认提供 Markdown 编辑器，但后续会在当前的富文本编辑器支持编写 Markdown，也可以通过插件提供单独的 Markdown 编辑器。
- 暂时仅支持 H2 Database 和 PostgreSQL，MySQL 暂时无法支持，详情可查阅 <https://github.com/halo-dev/halo/issues/2464>。
- 后台管理入口修改为 `/console`。

## 文档

<https://docs.halo.run/2.0.0-SNAPSHOT>

## 资源

适用于 2.0 Alpha 的插件和主题可查阅：<https://github.com/halo-sigs/awesome-halo>

<hr /><em>This discussion was created from the release <a href='https://github.com/halo-dev/halo/releases/tag/v2.0.0-alpha.1'>2.0.0-alpha.1</a>.</em>

**Comment by liuzhihang** [comment]
WIP

**Comment by ruibaby** [comment]
【直播回放】20221008 Halo v2.0 新版本预热说明会 https://www.bilibili.com/video/BV1MR4y1R7AK/?spm_id_from=333.999.0.0&vd_source=141cce00579ee38f0bd0757cf88a01ff

**Reply by shiker1996** [reply]
还有直播？咋没直播预告！！👊🏿👊🏿👊🏿

**Reply by ruibaby** [reply]
忘了在 GitHub 发...

你可以看看回放，下次我们会在 Issue 或者 Discussion 预告。
